lib/govuk_app_config/govuk_error/configuration.rb

```diff
@@ -25,8 +25,13 @@ def should_capture=(closure)
     def ignore_excluded_exceptions_in_data_sync
       lambda { |error_or_event|
         data_sync_ignored_error = data_sync_excluded_exceptions.any? do |exception_to_ignore|
+          exception_to_ignore = Object.const_get(exception_to_ignore)
```

Comment:
For compatibility with the Raven interface this should be able to handle the actual class passed in. https://github.com/getsentry/sentry-ruby/blob/7a0cebc54c5508d379769c7af688c8218721d734/lib/raven/configuration.rb#L447

Reply:
This was quite simple to add, so I've tacked a commit on - but I do wonder how necessary it is, given `data_sync_excluded_exceptions` is our own construct, and the [Raven docs](https://docs.sentry.io/platforms/ruby/configuration/options/) seem to suggest passing string values to `excluded_exceptions` (the closest thing to what we've built).

Happy to remove the commit I've just added, or keep it - I don't feel strongly either way.

Reply:
Cool thanks - I think it's better that the APIs are compatible. It probably isn't absolutely necessary but does seem a common pattern where classes can be passed in directly or as a string, feels unusual to only accept string but not the actual constant.